src/runners/runnerScheduler.ts

```diff
@@ -26,14 +27,14 @@ class RunnerScheduler {
     private _javaHome: string;
     private _context: ExtensionContext;
     private _isRunning: boolean;
-    private _runnerMap: Map<ITestRunner, ITestItem[]> | undefined;
+    private _runnerMap: Map<BaseRunner, ITestItem[]> | undefined;
 
     public initialize(javaHome: string, context: ExtensionContext): void {
         this._javaHome = javaHome;
         this._context = context;
     }
 
-    public async run(testItems: ITestItem[], isDebug: boolean, node?: TestTreeNode): Promise<void> {
+    public async run(testItems: ITestItem[], isDebug: boolean, node?: TestTreeNode, launchConfiguration?: DebugConfiguration): Promise<void> {
```

Comment:
TestTreeNode is data model at ui layer, it's better not to use it directly in the business logic layer.